apps/include/apps.h

```diff
@@ -105,7 +105,7 @@ X509_REQ *load_csr(const char *file, int format, const char *desc);
 X509 *load_cert_pass(const char *uri, int maybe_stdin,
                      const char *pass, const char *desc);
```

Comment:
Hmmmm, are the certs ever passphrase protected?  Yes, I understand that the *keys* in a PKCS#12 object may be protected, but the certs...?  Mind you, I'm still confused by PKCS#12, so I'm ready for a surprise :wink:  However, if the certs can be extracted without a passphrase, it's possible that the the 'file:' loader needs a bit of tuning...

Reply:
I suppose the certs might not be passphrase protected in some special cases but usually IMO they are encrypted as well (although with stupidly weak encryption).

Reply:
Ah, it's really about the whole PKCS#12 file, not the included certificates...  of course.  Yeah ok.